dev-cpp/blitz/blitz-1.0.3_pre20200524.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR=emake
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit cmake python-any-r1
+
+DESCRIPTION="Multi-dimensional array library for C++"
+HOMEPAGE="https://github.com/blitzpp/blitz"
+
+COMMIT=39f885951a9b8b11f931f917935a16066a945056
```

Comment:
Standard structure has DESCRIPTION/HOMEPAGE/SRC_URI/S together, then a blank newline, then LICENSE/SLOT/KEYWORDS/IUSE/RESTRICT.

Reply:
I'd also quote COMMIT and put it above DESCRIPTION.